homeassistant/components/feedreader.py

```diff
@@ -31,25 +34,33 @@
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: {
         vol.Required(CONF_URLS): vol.All(cv.ensure_list, [cv.url]),
+        vol.Optional(CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL):
+            cv.time_period,
+        vol.Optional(CONF_MAX_ENTRIES, default=MAX_ENTRIES): cv.positive_int
```

Comment:
Could you rename the const `MAX_ENTRIES` to `DEFAULT_MAX_ENTRIES`?

Reply:
Sure, will change this now.